scripts/set-password.js

```diff
@@ -3,6 +3,7 @@ import { models } from '../src/server/models'
 
 function printHelp() {
   console.log('set-password.js <email of the user to set password> <password>')
+  process.exit(1)
```

Comment:
Since this function now exits the process, maybe it should be renamed `printHelpAndExit` or `exitWithHelpMessage` or something like that to reflect the behavior

Reply:
Done

Reply:
Done